apps/backend/src/database/database.service.ts

```diff
@@ -12,9 +12,9 @@ export class DatabaseService {
   }
 
   async cleanAll(): Promise<void> {
-    Object.values(this.sequelize.models).forEach((model) => {
-      model.destroy({where: {}});
-    });
+    for (const model of Object.values(this.sequelize.models)) {
+      await model.destroy({where: {}});
```

Comment:
instead of awaiting every iteration within this loop, it is better to make a collection of all of the promises, make them a single promise with promise.all, and then await that.

Reply:
Added Promises.all to manage collection of promises.